src/App.js

```diff
@@ -82,16 +83,40 @@ const getListStyle = (isDraggingOver, style = {}) => ({
 
 const usePersistedState = (key, initialState) => createPersistedState(key)(initialState);
 
+const defaultRowData = {
+  developer: false,
+  list: [],
+  style: { flexWrap: 'wrap' },
+};
+
+const addRowForDev = developerName => ({
+  ...defaultRowData,
+  id: nanoid(),
+  name: developerName,
+  developer: true,
+});
+
+const developerNames = [
```

Comment:
dev names shouldn't be static as Secil is using this for different teams. once a session is saved to local storage, it is enough to restore it back with saved devs.

and clear button should only remove tasks not devs so that we don't need to reenter devs again.